immuni_analytics/core/managers.py

```diff
@@ -48,12 +48,14 @@ def analytics_redis(self) -> Redis:
             raise RuntimeError("Attempting to use Analytics redis before initialization")
         return self._analytics_redis
 
-    async def initialize(self) -> None:
+    async def initialize(self, initialize_mongo: bool = False) -> None:
```

Comment:
What should we do with `immuni_common` base class method definition? Are we forced to change it (in common) to `async def initialize(self, **kwargs: Any) -> None`?

Reply:
I would say so 👍